app.js

```diff
@@ -93,11 +93,10 @@ async function hasCheckmarkReaction({ client, channel, timestamp }) {
 }
 
 async function processThreadMessagesForGratitude(client, event) {
-  if (event.user !== event.parent_user_id) return;
   if (await hasCheckmarkReaction({ client, channel: event.channel, timestamp: event.thread_ts })) return;
 
   const text = event.text.toLowerCase();
-  if (text === "solved") {
+  if (text === "solved" || text.endsWith(" has been updated to `done`.")) {
```

Comment:
Does it care about casing, for `Done`?

![Screenshot 2024-10-02 at 11 04 21 AM](https://github.com/user-attachments/assets/5e7424d1-20ab-40d3-a679-602605beefeb)

Reply:
the comparison is case insensitive. the text gets converted to lower case before `const text = event.text.toLowerCase();`

Reply:
oh nice! missed that top line! 👍